cmd/util/nodehelpers/logging.go

```diff
@@ -0,0 +1,104 @@
+package nodehelpers
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+var globalFileHandlerFactory = fileHandlerFactory{}
+
+type fileHandlerFactory struct {
+	writer  *lumberjack.Logger
+	records chan *log.Record
+	cancel  context.CancelFunc
+}
+
+// newHandler is not threadsafe
+func (l *fileHandlerFactory) newHandler(logFormat log.Format, config *genericconf.FileLoggingConfig, pathResolver func(string) string) log.Handler {
```

Comment:
This seems to belong in genericconf.. or anyway, FileLoggingConfig and this logger seem to belong in the same package. possibly entire nodehelpers/genericconfig should be merged (not sure)